index.mjs

```diff
@@ -91,8 +92,8 @@ class WordWrap {
 
   /**
    * Wraps the input text, returning an array of strings (lines).
-   * @param {string} - input text
-   * @param {object} - Accepts same options as constructor.
+   * @param {string} text input text
```

Comment:
"text input text"?

Reply:
The first word `text` is the parameter name
https://jsdoc.app/tags-param.html

<img src="https://user-images.githubusercontent.com/1148376/129360600-7836ba6a-3bac-4c07-9669-c17dfac0a192.png" width="500">

This is how it looks like in my editor
<img src="https://user-images.githubusercontent.com/1148376/129360872-dc786d3d-103b-4d73-9b6b-9171a95a0a39.png" width="500">

Reply:
OK.. obviously, it's not safe to assume we all use the same editor and have the same syntax highlighting.. I think it's more clear to use the `-` separator, so `text - Input text` would remove any confusion.. this is valid jsdoc syntax.

Reply:
what editor do you use? just curious

Reply:
Sublime and vim.. (but even that is irrelevant because syntax highlighting is configurable - i could use a hand-rolled theme or just switch if off altogether)

Reply:
added -